client/src/modules/authorization/model/actions.js

```diff
@@ -113,12 +113,14 @@ export const resetPassword = email => dispatch =>
     });
 
 export const updatePassword = (token, password, passwordConfirmation) =>
-  postData(`/auth/recovery-password/${token}`, {
+  postData(`/auth/update-password/${token}`, {
     password,
     passwordConfirmation
   })
     .then(() => history.replace('/password-recovery-success'))
     .catch(err => {
-      // TODO: IF err instanceof BadRequestException redirect to 'LinkExpired' component
       throw err;
     });
+
+export const resendRecoveryLink = token =>
+  postData(`/auth/resend-recovery-link/${token}`);
```

Comment:
maybe if this is post send the token in request body instead of in the URL?

Reply:
why not in url ?

Reply:
leave as it is. This more logical for me. When we send data via POST we usually send it in the request body, except listId and cohortId...